doc/BENCHMARKING.md

```diff
@@ -39,10 +39,17 @@ You can use the following steps.
         export S3_BUCKET_BENCH_FILE=bench_file_name
         export S3_BUCKET_SMALL_BENCH_FILE=small_bench_file_name
 
-3. Create the bench files manually in your bucket. The size of the files must be exactly the same as the size defined in fio configuration files. The easiest way to do this is running fio against your local file system first to let fio create the files for you, and then upload them to your S3 bucket using the AWS CLI. For example:
-
-        fio --directory=your_local_dir --filename=your_file_name mountpoint-s3/scripts/fio/read/seq_read_small.fio
-        aws s3 cp your_local_dir/your_file_name s3://${S3_BUCKET_NAME}/${S3_BUCKET_TEST_PREFIX}
+3. Create the bench files manually in your S3 bucket. You can do that by mounting the bucket on your machine using Mountpoint. Then, running fio jobs against your mount directory to let fio create the files for you.
+
+        mount-s3 DOC-EXAMPLE-BUCKET path/to/mount/ --prefix benchmark/ --part-size=16777216
+        cd mountpoint-s3/scripts/fio/
+        fio --directory=path/to/mount/ --filename=bench5MB.bin read/seq_read_small.fio --create-only=1
```

Comment:
Does this really work? I thought job options are supposed to come before job file. Same for other commands.
```suggestion
        fio --directory=path/to/mount/ --filename=bench5MB.bin --create-only=1 read/seq_read_small.fio
```

Reply:
`create_only=1` works with it being after the job file. I just tried it for small files it does not run other jobs and just create the file.

Reply:
nit: still other options are the the front. we should group them together, I think.